pom.client.xml

```diff
@@ -815,8 +823,8 @@
                 --add-opens com.azure.core/com.azure.core.implementation.entities=com.fasterxml.jackson.databind
                 --add-opens com.azure.core/com.azure.core.implementation.entities=ALL-UNNAMED
               </argLine>
-              <useSystemClassLoader>false</useSystemClassLoader>
-              <forkCount>1</forkCount>
+              <useSystemClassLoader>true</useSystemClassLoader>
+<!--              <forkCount>1</forkCount>-->
```

Comment:
This used to break some of the storage tests if the forkCount is not set to 1. Will this work for storage tests now?